Week-1/Blastoise/Web_scrapper.py

```diff
@@ -0,0 +1,127 @@
+import requests
```

Comment:
- There's a bunch of Bad-Whitespace issues in your code. Please consider using a linter in your editor.
- Adding module docstring is recommended. Refer: https://www.python.org/dev/peps/pep-0257/

Reply:
I was using VS Code linter and program was working fine without any indentation errors.

Reply:
https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements

Reply:
ook got it.Thanks